openedx/openedx-platform
# refactor: modified the imports to use openedx-events library

## Description
This PR modified the imports to use openedx-events library, it's due to the extraction of the `COURSE_DISCUSSIONS_CHANGED` event definition and his `attr` data was move from `edx-platform` to `openedx-events`.

## Supporting information
Discussion event extraction PR
- https://github.com/eduNEXT/openedx-events/pull/45

Related PR
- https://github.com/openedx/edx-platform/pull/29082
- https://github.com/openedx/edx-platform/pull/28749

## Testing instructions

1. In studio, modified the content of a course in a unit and publish changes.
2. You should see new DiscussionTopicLink objects.
3. In studio logs you can see log.info for "Updating existing discussion topic links for" or "Creating new discussion topic links for".

You can use the instructions from PR <https://github.com/openedx/edx-platform/pull/29082> too.

## Deadline

"None" 

## Other information

- https://github.com/openedx/edx-platform/pull/28864
